libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/AnalyticsPublisherService.java

```diff
@@ -59,11 +52,11 @@ public AnalyticsPublisherService() {
     public static void startActionPublish(Context context) {
         final Intent intent = new Intent(context, AnalyticsPublisherService.class);
         intent.setAction(ACTION_PUBLISH);
-        context.startService(intent);
+        enqueueWork(context, AnalyticsPublisherService.class, JOB_ID, intent);
     }
 
     @Override
-    protected void onHandleIntent(Intent intent) {
+    protected void onHandleWork(Intent intent) {
```

Comment:
`JobIntentService` has an abstract method called `onHandleWork` that does the work (like `onHandleIntent` in an `IntentService`).